spec/scrum_lint/runner_spec.rb

```diff
@@ -15,17 +20,40 @@ module ScrumLint
     it 'validates the selected board' do
       allow(Linter::MissingTaskList).to receive(:call).and_call_original
       allow(Linter::ExtraList).to receive(:call).and_call_original
+      allow(Linter::MissingTaskList).to receive(:call).and_call_original
       runner.([])
       expect(Linter::MissingTaskList).to have_received(:call)
         .with(instance_of(Board))
       expect(Linter::ExtraList).to have_received(:call).with(instance_of(Board))
+      expect(Linter::ExtraList).to have_received(:call).with(instance_of(Board))
```

Comment:
Added code coverage! 🎉

Reply:
does having this line here twice mean we expect this method to be called twice? if so i think you can do `have_received(:call).twice.with(..)`

Reply:
Haha nope. It means I wasn't very careful with my copy/pasta-ing.